packages/api/src/useCases/cases/fetchCasesAndFlitersTests/filters/resolvedCaseDataRange.e2e.test.ts

```diff
@@ -0,0 +1,124 @@
+import type { User } from "@moj-bichard7/common/types/User"
+import type { FastifyInstance } from "fastify"
+
+import type { CaseIndexQuerystring } from "../../../../types/CaseIndexQuerystring"
+
+import { createCases } from "../../../../tests/helpers/caseHelper"
+import { SetupAppEnd2EndHelper } from "../../../../tests/helpers/setupAppEnd2EndHelper"
+import { createUsers } from "../../../../tests/helpers/userHelper"
+import { Reason } from "../../../../types/CaseIndexQuerystring"
+import { ResolutionStatus, ResolutionStatusNumber } from "../../../dto/convertResolutionStatus"
+import { fetchCasesAndFilter } from "../../fetchCasesAndFilter"
+
+describe("fetchCasesAndFilter filtering by court date e2e", () => {
+  let helper: SetupAppEnd2EndHelper
+  let app: FastifyInstance
+  let user: User
+
+  const defaultQuery: CaseIndexQuerystring = {
+    caseState: ResolutionStatus.Resolved,
+    maxPerPage: 25,
+    pageNum: 1,
+    reason: Reason.All
+  }
+  const firstDate = new Date("2001-09-26")
+  const secondDate = new Date("2008-01-26")
+  const thirdDate = new Date("2008-03-26")
+  const fourthDate = new Date("2013-10-16")
+
+  beforeAll(async () => {
+    helper = await SetupAppEnd2EndHelper.setup()
+    app = helper.app
+    helper.postgres.forceIds = [1]
+
+    await helper.postgres.clearDb()
+    await helper.dynamo.clearDynamo()
+
+    user = (await createUsers(helper.postgres, 1))[0]
+    await createCases(helper.postgres, 4, {
+      0: {
+        court_date: firstDate,
+        error_resolved_by: user.username,
+        error_status: ResolutionStatusNumber.Resolved,
+        resolution_ts: firstDate
+      },
+      1: {
+        court_date: secondDate,
+        error_resolved_by: user.username,
+        error_status: ResolutionStatusNumber.Resolved,
+        resolution_ts: secondDate
+      },
+      2: {
+        court_date: thirdDate,
+        error_resolved_by: user.username,
+        error_status: ResolutionStatusNumber.Resolved,
```

Comment:
Should this be unresolved or is "unresolvedness" just indicated by `resolution_ts` being null?

Reply:
I've added this for clarity:

```typescript
court_date: thirdDate,
error_resolved_by: user.username,
error_status: ResolutionStatusNumber.Resolved,
resolution_ts: null,
trigger_status: ResolutionStatusNumber.Unresolved
```